src/ansible_navigator/data/image_introspect.py

```diff
@@ -411,7 +420,10 @@ def main():
             response[result_as_dict["__id_"]] = result_as_dict
     except Exception as exc:  # noqa: BLE001
         response["errors"].append(str(exc))
-    print(json.dumps(response))
+    if serialize:
+        print(json.dumps(response))
```

Comment:
remove this statement, is for debugging purposes

Reply:
Removing this statement will result in failure while collecting execution environment information, hence keeping it there for now.